Source/DafnyCore/ParserErrorDetail.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright by the contributors to the Dafny Project
+// SPDX-License-Identifier: MIT
+
+using System;
+using System.Numerics;
+using System.Globalization;
+
+namespace Microsoft.Dafny;
+
+public class ParserErrorDetail {
+
+  public static string p_bad_const_initialize_op = "p_bad_const_initialize_op";
```

Comment:
To avoid having to repeat the name in the strong, you can consider using an enum
https://stackoverflow.com/a/71090888/1287856